packages/marketplace/src/components/ui/app-revision-comparision/app-revision-comparision.tsx

```diff
@@ -118,23 +120,46 @@ export const AppRevisionComparision: React.FC<AppRevisionComparisionProps> = ({
             </div>
           )
         }
-        if (['redirectUris', 'signoutUris', 'limitToClientIds'].includes(key)) {
-          const currentString = Array.isArray(app[key]) ? app[key].join(' ') : ''
-          const changedString = Array.isArray(revision[key]) ? revision[key].join(' ') : ''
+        if (key === 'desktopIntegrationTypeIds') {
+          const oldIntegrationTypeArray =
+            app.desktopIntegrationTypeIds?.map(
+              id =>
+                desktopIntegrationTypes.data?.find((integration: DesktopIntegrationTypeModel) => integration.id === id)
```

Comment:
I think should move this logic to others function. Because reader don't know what's need to do here